ecodse_funtime_alpha/data.py

```diff
@@ -56,7 +56,7 @@ def get_dataset(image_dir, labels_csv):
 
     img_label_ds = tf.data.Dataset.zip((img_ds, label_ds))
 
-    return img_label_ds
+    return img_label_ds, len(samples)
```

Comment:
No, we should call tf.data.experimental.cardinality

Reply:
removed from this branch, will implement in a new branch